DockerComposeFixture/DockerFixture.cs

```diff
@@ -32,9 +32,9 @@ public DockerFixture(IMessageSink output)
         /// If you call this multiple times on the same DockerFixture then it will be ignored.
         /// </summary>
         /// <param name="setupOptions">Options that control how docker-compose is executed.</param>
-        public void InitOnce(Func<IDockerFixtureOptions> setupOptions)
+        public async Task InitOnceAsync(Func<IDockerFixtureOptions> setupOptions)
```

Comment:
Whilst I have no problem with this being async, just remember that as a fixture people will be calling this from a constructor (which are inherently synchronous). So they will have to do a .GetAwaiter().GetResult(). However if you are firing off multiple async Tasks then this would still make sense.